torch_xla/csrc/ops/dynamic_ir.cpp

```diff
@@ -41,6 +41,9 @@ int64_t SizeNode::getDynamicValue() const {
   }
   torch::lazy::NodePtr cloned =
       torch::lazy::MakeNode<SizeNode>(operands_[0], dim_);
+  // Wrap the IR of SizeNode into a dummy tensor and execute/fetch the value
```

Comment:
By "IR of SizeNode", it seems you mean the variable `cloned` which is a `torch::lazy::NodePtr` object. I thought `torch::lazy::NodePtr` denote the SizeNode. So I'm a bit confused. What's the difference between the node and IR of the node?